conductr_cli/conduct_info_inspect.py

```diff
@@ -206,6 +210,32 @@ def display_title_table(title):
     log.screen(title_separator)
 
 
+def get_uptime(start_time):
+    intervals = (
+        ('d', 86400),  # 60 * 60 * 24
+        ('h', 3600),   # 60 * 60
+        ('m', 60),     # 60
+        ('s', 1),
+    )
+
+    def humanize(timedelta):
+        seconds = timedelta.days * 86400 + timedelta.seconds
+        result = []
+        for name, count in intervals:
+            value = seconds // count
+            if value:
+                seconds -= value * count
+                # if value > 0:
+                if value == 1:
+                    name = name.rstrip('s')
+                result.append("{}{}".format(value, name))
+        return ', '.join(result)
+
+    start_time_local = arrow.get(start_time).to('local')
+    current_time_local = arrow.now().to('local')
```

Comment:
I think we need to do the seconds calculation with UTC. Local timezones don't necessary have 86400 seconds in them due to DST but POSIX time always does. Also there's the consideration of clocks being set incorrectly, we probably want to make sure we don't show a negative uptime value.

Perhaps `seconds = max(0, arrow.now().timestamp - arrow.get(start_time).timestamp)`